### ISSUE TITLE
Only getting 25 items from IP list

### ISSUE THREAD

**Issue by menghoumenghou** [cloudflare/python-cloudflare#155]
Followed the example code to pull the IPs from one of the IP lists, however I can only get 25 items, I am guessing there is a pagination mechanism.  

        print('EXISTING LIST LOOKS LIKE:')
        items = cf.accounts.rules.lists.items(account_id, list_id)
        for item in items:
            print('%s %s %s %-30s ; %s' % (item['id'], item['created_on'], item['modified_on'], item['ip'], item['comment']))
        print('')

Please see if this can be fixed. Thanks.

**Comment by mahtin** [comment]
The `params` argument can control paging thru long lists. This is documented on the API description page. Here's a very simple example:
```
    items = cf.accounts.rules.lists.items(account_id, list_id), params={'per_page':400})
```
The `per_page` parameter is documented for many API calls that return potentially lots of items. 

Hope that helps.

**Comment by mahtin** [comment]
I should add that `params` done a lot more than just paging. It's very API call specific.

**Comment by menghoumenghou** [comment]
Tried to set the above per_page parameter to 100 but I am still only getting 25 items back. 
and I found the documentation regarding this:
https://api.cloudflare.com/#lists-get-list-items
apparently there is a cursor being sent back in the response, and you will need to request with that cursor value to get the next batch of result.

And yes, that is very API call specific as some of the APIs are still using per_page parameter to control how many items are returned.
